BuildPropertiesPlugin/build.gradle

```diff
@@ -1,7 +1,7 @@
 ext {
   isLocalDeploy = !project.hasProperty('maven')
   pluginGroup = 'com.novoda.build-properties'
-  pluginVersion = isLocalDeploy ? 'LOCAL' : '0.9.0'
+  pluginVersion = isLocalDeploy ? 'LOCAL' : '0.9.5'
```

Comment:
could do both I guess. I like that this way `pluginVersion` is in the root extension, therefore accessible from the buildscript of every subproject